src/main/scala/io/iohk/ethereum/db/storage/CodeStorage.scala

```diff
@@ -0,0 +1,22 @@
+package io.iohk.ethereum.db.storage
+
+import akka.util.ByteString
+import io.iohk.ethereum.db.dataSource.DataSource
+
+import CodeStorage._
+
+class CodeStorage(val dataSource: DataSource) extends KeyValueStorage[CodeHash, Code] {
+  type T = CodeStorage
```

Comment:
Yes, this is required by the `KeyValueStorage` trait as it is used in `def update(toRemove: Seq[K], toUpdate: Seq[(K, V)]): T` and other functions that return a modified `KeyValueStorage`. This is also done in `MptNodeStorage` and the other different `KeyValueStorage` implementations.